android-json-form-wizard/src/test/java/com/vijay/jsonwizard/widgets/EditTextFactoryTest.java

```diff
@@ -226,6 +228,51 @@ public void testNameWithCumulativeTotalValidatorEditTextFactoryInstantiatesViews
         Assert.assertEquals(1, viewList.size());
     }
 
+    @Test
+    public void testCatchJsonExceptionViewsShouldBeEmpty() throws Exception {
+        Assert.assertNotNull(factory);
+        EditTextFactory factorySpy = Mockito.spy(factory);
+        Assert.assertNotNull(factorySpy);
+
+        FormUtils formUtils = new FormUtils();
+        FormUtils formUtilsSpy = Mockito.spy(formUtils);
+        Assert.assertNotNull(formUtilsSpy);
+
+        Mockito.doReturn(resources).when(context).getResources();
+        Assert.assertNotNull(resources);
+
+        context.setTheme(R.style.NativeFormsAppTheme);
+        Mockito.doReturn(rootLayout).when(factorySpy).getRelativeLayout(context);
+        Assert.assertNotNull(rootLayout);
+
+        Mockito.doReturn(editTextLayout).when(rootLayout).findViewById(R.id.edit_text_layout);
+        Assert.assertNotNull(editTextLayout);
+
+        Mockito.doReturn(editText).when(editTextLayout).findViewById(R.id.edit_text);
```

Comment:
this is a robolectric test, why do we need to mock all these views?

Reply:
maintain the consistency, other tests in this class also have this pattern.

Reply:
you can change  this to a robolectric test.

Reply:
changed.